src/Microsoft.AspNetCore.Mvc.Core/ApiBehaviorOptions.cs

```diff
@@ -28,10 +28,21 @@ public class ApiBehaviorOptions
         }
 
         /// <summary>
-        /// Disables the filter that returns an <see cref="BadRequestObjectResult"/> when
-        /// <see cref="ActionContext.ModelState"/> is invalid. 
+        /// Gets or sets a value that determines if the filter that returns an <see cref="BadRequestObjectResult"/> when
+        /// <see cref="ActionContext.ModelState"/> is invalid. Defaults to <c>true</c>.
         /// <seealso cref="InvalidModelStateResponseFactory"/>.
         /// </summary>
         public bool EnableModelStateInvalidFilter { get; set; } = true;
```

Comment:
Sorry I didn't catch this in the past, this should be called `SuppressModelStateInvalidFilter` and default to false. We want boolean options to default to false, it's really surprising when something has a non-default value.